helpers/event/event_slack_notifier.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import celery
+from config import Config
+
+"""
+This function will call slack bot url
+"""
+
+@celery.task(name="notify-slack")
+def notify(event_Id):
```

Comment:
Please refer to variable naming conventions for `event_Id`. (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)